Sources/MQ/Errors/Unimplemented.swift

```diff
@@ -87,6 +87,93 @@ public func unimplemented<R>(
 	}
 }
 
+/// Convenient placeholder for unimplemented function.
+///
+/// Placeholder for a function indicating unimplemented part of code.
+/// Terminates process with ``Unimplemented`` error as the cause when the function becomes executed.
+///
+/// - Parameters:
+///   - message: Message associated with this error.
+///   Default value is "Unimplemented".
+///   - file: Source code file identifier.
+///   Filled automatically based on compile time constants.
+///   - line: Line in given source code file.
+///   Filled automatically based on compile time constants.
+/// - Returns: Placeholder function which terminates the process with ``Unimplemented`` error when executed.
+public func unimplementedAsync<R>(
+	_ message: StaticString = "Unimplemented",
+	file: StaticString = #fileID,
+	line: UInt = #line
+) -> () async -> R {
+	{
+		Unimplemented
+			.error(
+				message: message,
+				file: file,
+				line: line
+			)
+			.asFatalError()
+	}
+}
+
+/// Convenient placeholder for unimplemented function.
+///
+/// Placeholder for a function indicating unimplemented throwing part of code.
+/// Throws ``Unimplemented`` when executed.
```

Comment:
Please change a little bit more from "Placeholder for a function indicating unimplemented throwing part of code." to "Placeholder for a function indicating unimplemented part of code.".
And for next line please change from "Throws ``Unimplemented`` when executed." to "Throws ``Unimplemented`` when the function becomes executed."
It applies for all throwing functions.

Reply:
OK